### ISSUE TITLE
Usage with webpack noParse

### ISSUE THREAD

**Issue by R1ZZU** [twitter/twemoji#198]
Bug with webpack `noParse` option:

```
twemoji.npm.js:1 Uncaught ReferenceError: global is not defined
    at Object.23 (twemoji.npm.js:1)
```

Workaround:
```js
require('imports?global=>window!twemoji');
```

Fix proposal:
```diff
- var location = global.location || {};
+ var location = (global || window).location || {};
```

**Comment by WebReflection** [comment]
Nope, that doesn't work. If it does it's a webpack bug. You cannot reference something directly if not defined already so that OR is a reference error if global is not available.

**Comment by R1ZZU** [comment]
> it's a webpack bug

It's a runtime behaviour, webpack has nothing to do with it 😏 

just to add more details: `noParse` option just drops code in final bundle as-is, so this code behaves as if it is included with `<script>` tag. With reference error you're right, in strict this code will throw a reference error... In this case my proposal is wrong, but issue is still legit.

**Comment by WebReflection** [comment]
> It's a runtime behaviour, webpack has nothing to do with it

it's broken code. Write this in console:

```js
var failure = nope || window;
```

And admire the error. So, that code is broken and won't land in the library because `global` doesn't exist on the browser.

Please note this has nothing to do with strict mode, this is how ES has always been.
